### PR TITLE
IMPROVE THE BACKGROUND OF SPELLING BEE GAME

### LINKED ISSUE #4163: [Enhancement]: IMPROVE THE BACKGROUND OF SPELLING BEE GAME
### Do you want to have the enhancement of existing game ? 😀 Describe yourself..

Game enhancement
The background of spelling bee game is very simple,.

### Describe the solution you'd like

Solution steps
I will make the background of spelling bee game attractive.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by TusharChamlikar**
## PR Description 📜

Fixes #4163
the background of spelling bee game was simple, i made it attractive by changing background.
<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
Screenshot 2024-06-03 194225

<hr>

## Add your screenshots(Optional) 📸

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by TusharChamlikar** [comment]
@kunjgit please merge my request sir

**Comment by TusharChamlikar** [comment]
@kunjgit

**Comment by TusharChamlikar** [comment]
@kunjgit sir plz merge my request

**Comment by TusharChamlikar** [comment]
@kunjgit

File: Games/Chrome_Dinosaur_Game/dino.jpg
Comment (ishita-43) [inline Games/Chrome_Dinosaur_Game/dino.jpg:1]: Remove file if not needed!!

File: Games/Chrome_Dinosaur_Game/th.jpeg
Comment (ishita-43) [inline Games/Chrome_Dinosaur_Game/th.jpeg:1]: Remove if not impacting the results!!

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Make clear PR only making changes in desired game and not tempering with others!!
Hoping you will make changes soon!!

**Comment by kunjgit** [comment]
Thank you  @TusharChamlikar , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request closed without merging**